examples/deploy/addon/helloworld-template/resources/addon_template.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: AddOnTemplate
+metadata:
+  name: hello-template
+spec:
+  addonName: hello-template
+  agentSpec:
+      workload:
+        manifests:
+          - kind: Deployment
+            apiVersion: apps/v1
+            metadata:
+              name: hello-template-agent
+              namespace: open-cluster-management-agent-addon
+              annotations:
+                "addon.open-cluster-management.io/deletion-orphan": ""
```

Comment:
the annotation deletion-orphan still can work in addonTemplate?

Reply:
yes, it can work since the template controller just start a go routine to run the framework code.